package/yast2-registration.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Mar  2 12:54:20 UTC 2017 - lslezak@suse.cz
+
+- Fixed layout of the "Extension and Module Selection" dialog
+  in text mode to make all items visible (bsc#1024464)
```

Comment:
This is fine, but in"Fixed layout of the ... dialog", "fixed" is an adjective (Pevné rozvržení dialogu) whereas we want "Fixed **the** layout of the ... dialog".